Idref.js

```diff
@@ -62,6 +64,8 @@ function getSearchResults(doc, checkOnly) {
 
 function doWeb(doc, url)
 {
+	searchResultsReady = getSearchResults(doc, true);
```

Comment:
That is not further used and should therefore been deleted.

Reply:
Fixed.